internal-tooling/README.md

```diff
@@ -0,0 +1,44 @@
+# nodejs-storage benchmarking
+
+**This is not an officially supported Google product**
```

Comment:
Change `This is not an officially supported Google product` to `This is not a supported Google product`. It's not unofficially supported either.